src/Diagnostics/FxCop/System.Runtime.Analyzers/Core/Design/DefineAccessorsForAttributeArguments.Fixer.cs

```diff
@@ -103,10 +108,19 @@ public class DefineAccessorsForAttributeArgumentsFixer : CodeFixProvider
             return symbolEditor.GetChangedDocuments().First();
         }
 
-        private async Task<Document> MakePublic(Document document, SyntaxNode getMethod, CancellationToken cancellationToken)
+        private async Task<Document> MakePublic(Document document, SyntaxNode getMethod, SyntaxNode property, CancellationToken cancellationToken)
         {
+            // Clear the accessibility on the getter.
             DocumentEditor editor = await DocumentEditor.CreateAsync(document, cancellationToken).ConfigureAwait(false);
-            editor.SetAccessibility(getMethod, Accessibility.Public);
+            editor.SetAccessibility(getMethod, Accessibility.NotApplicable);
+
+            // If the containing property is not public, make it so
+            var propertyAccessibility = editor.Generator.GetAccessibility(property);
+            if (propertyAccessibility != Accessibility.Public)
+            {
+                editor.SetAccessibility(property, Accessibility.Public);
+            }
```

Comment:
As it stands, if we don't fix the setter, it would get squiggled and the user would get a fix to fix up the accessibility but I agree we should just that do that in one fix. I'll fix that.